corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -25,6 +25,11 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
                 options,
                 menus;
 
+                FormplayerFrontend.durationMetToShowLoading = false;
+                let intervalId = setInterval(() => {
```

Comment:
This can be `const` unless it gets redefined in the same scope.

Reply:
delay functionality will be handled in a separate PR https://github.com/dimagi/commcare-hq/commit/7114ec7e574e361b83da29fb77cadb887debca81